### DISCUSSION TITLE
240323

### DISCUSSION THREAD

**Discussion by jisunipark** [codeit-part4-8team-project/main#70] [팀회의]

**Comment by jisunipark** [comment]
## 논의사항

## 어제 한 일
- 팀 페이지 탭 컨텍스트 -> 하루 종일 이거만 봤는데 답을 못 찾았습니다

## 오늘 할 일
- api 연동 (계속 미뤄지는군요 ㅎㅎ..)

## 오늘의 일정 (코어 타임)

## 멘토님 질문 사항

**Comment by caseV12** [comment]
## 논의사항

## 어제 한 일
user context 작성
## 오늘 할 일
하던 거 완성
## 오늘의 일정 (코어 타임)
x
## 멘토님 질문 사항
